service/src/test/java/bio/terra/drshub/tracking/TrackingInterceptorTest.java

```diff
@@ -194,18 +194,24 @@ void testHappyPathEmittingToBardWithResolvedCloudAzure() throws Exception {
   }
 
   @Test
-  void testHappyPathEmittingToBardWithTransactionId() throws Exception {
+  void testHappyPathEmittingToBardWithFileName() throws Exception {
```

Comment:
I removed the `testHappyPathEmittingToBardWithTransactionId` because I realized I already mocked the transactionId in the `setUp` and the `userLoggingMetrics` is autowired so I do not need to set the transactionId property in the test. So after removing those unnecessary lines it looks exactly like `testHappyPathEmittingToBard`

I added `testHappyPathEmittingToBardWithFileName` too so the diff looks a bit confusing here.